### PR TITLE
Bump up code coverage to 100% for a few more files.

### PR DESCRIPTION
**Pull request by geeknoid**
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/146)

### CODE REVIEW

File: adapter/ipListChecker/config_proto [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/google/home/mtail/go/src/istio.io/mixer/bazel-genfiles/adapter/ipListChecker
```
Comment (douglas-reid) [inline adapter/ipListChecker/config_proto]: we don't want to check this in, right?

Comment (geeknoid) [inline adapter/ipListChecker/config_proto]: Whoops. Removed.

PTAL.

File: pkg/adaptertesting/BUILD [hunk 2]
```diff
@@ -4,6 +4,15 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
...
+        "adapters.go",
+        "builders.go",
+    ],
+    deps = [
+        "//pkg/adapter:go_default_library",
+        "//pkg/aspect:go_default_library",
+        "//pkg/aspect/denyChecker:go_default_library",
+        "//pkg/aspect/listChecker:go_default_library",
+        "//pkg/registry:go_default_library",
+    ],
```
Comment (douglas-reid) [inline pkg/adaptertesting/BUILD]: can we add a 'testonly' attribute to this library while we are here? i don't think we want this code used outside of tests.

File: pkg/adaptertesting/adapters.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+func (r *fakeRegistrar) RegisterDeny(denyChecker.Adapter) error {
+	r.registrations++
+	return nil
+}
+
+// TestAdapterInvariants ensures that adapters implement expected semantics.
+func TestAdapterInvariants(a aspect.Adapter, r RegisterFunc, t *testing.T) {
+	if a.Name() == "" {
+		t.Error("All adapters need names")
```
Comment (douglas-reid) [inline pkg/adaptertesting/adapters.go]: For these tests, can we start the error log line with something indicating the method and/or adapter?

Maybe something like:
"Name() => "", all adapters need names"
"Description() => "", all adapters need descriptions"
"ValidateConfig(%T) => unexpected error for DefaultConfig(): %v", c, err
"Close() => unexpected error: %v", err
"RegisterFunc() => unexpected error: %v", err
"RegisterFunc() => %d registrations, want at least 1", fr.registrations
?

**Comment by geeknoid** [comment]
Done. PTAL.

**Comment by geeknoid** [comment]
I did add a testonly flag in adaptertesting...

**Pull request merged**